Cargo.toml

```diff
@@ -46,6 +46,9 @@ incrementalmerkletree = "0.2"
 # Developer tooling dependencies
 plotters = { version = "0.3.0", optional = true }
 
+[patch.crates-io]
+incrementalmerkletree = { git = "https://github.com/zcash/incrementalmerkletree.git", rev = "33b9bc9667a6a649d8ef6fbec4c54a79a851f3e6" }
```

Comment:
This crate bump includes an API change that is causing a failure in proptest code:
```
error[E0061]: this function takes 2 arguments but 1 argument was supplied
   --> src/tree.rs:298:26
    |
298 |                     tree.authentication_path(&leaf),
    |                          ^^^^^^^^^^^^^^^^^^^ ----- supplied 1 argument
    |                          |
    |                          expected 2 arguments
    |
note: associated function defined here
   --> /home/runner/.cargo/git/checkouts/incrementalmerkletree-cc507a7e80715a08/33b9bc9/src/lib.rs:257:8
    |
257 |     fn authentication_path(&self, position: Position, value: &H) -> Option<Vec<H>>;
    |        ^^^^^^^^^^^^^^^^^^^
```

Reply:
That failure is stale; github isn't correctly reporting build statuses at the moment.